### PR TITLE
Prepare 23.2.0

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def read_file(filename):
 
 setup(
     name='kinto-dist',
-    version='23.1.2',
+    version='23.2.0.dev0',
```
Comment (smarnach) [inline setup.py:26]: Is the "dev0" part here intentional?

Comment (leplatrem) [reply setup.py]: I hadn't seen that. I pushed a fix to re-prepare

**Review by smarnach — APPROVED** [review]

**Comment by leplatrem** [comment]
bors merge

**Pull request merged**